lib/MetaCPAN/Web/Controller/Source.pm

```diff
@@ -26,6 +26,7 @@ sub index : Path : Args {
             $c->model('API::Module')->get(@module),
         );
     }
+    die "Bad module supplied\n" if grep $_->{code} > 399, $source, $module;
```

Comment:
Should we capture the module name and response code here or is that happening somewhere else?

Reply:
No, it won't be captured anywhere. My thinking is the user will see the message, and act accordingly?